gradle/libs.versions.toml

```diff
@@ -11,7 +11,7 @@ apacheCommonsText = "org.apache.commons:commons-text:1.10.0"
 archunit = "com.tngtech.archunit:archunit:1.0.1"
 betterStrings = "com.antkorwin:better-strings:0.5"
 checker = { module = "org.checkerframework:checker", version.ref = "checker" }
-checker-qual = { module = "org.checkerframework:checker-qual", version.ref = "checker" }
+checker-qual = "org.checkerframework:checker-qual:3.29.0"
```

Comment:
I guess that's not the way it should be updated 🤔

Reply:
Whoops no indeed hmmm

Reply:
Lemme post this as a bug to the relevant OSS

Reply:
Huh there's already https://github.com/littlerobots/version-catalog-update-plugin/issues/89

Reply:
Maybe indeed [`checker-qual`](https://repo1.maven.org/maven2/org/checkerframework/checker-qual/3.30.0/) was available in Maven Central while [`checker`](https://repo1.maven.org/maven2/org/checkerframework/checker/3.30.0/) was still missing... both have the date around 8:30pm UTC y'day, but there might've been some eventual consistency glitch. Lemme check what happens if I run `./gradlew versionCatalogUpdate` locally now...

Reply:
Huh no, the same effect locally meh

Okay lemme just update it manually in the desired way